gateway-js/src/__tests__/loadSupergraphSdlFromStorage.test.ts

```diff
@@ -11,15 +11,26 @@ import {
   mockSupergraphSdlRequestIfAfterUnchanged,
 } from './integration/nockMocks';
 import mockedEnv from 'mocked-env';
+import nock from 'nock';
 
 describe('loadSupergraphSdlFromStorage', () => {
   let cleanUp: (() => void) | null = null;
 
-  afterAll(async () => {
+  beforeEach(() => {
+    if (!nock.isActive()) {
```

Comment:
I don't want to block merge on this if it turns out to be non-trivial, but I'll note that this is a fair amount of nock-related boilerplate, and it doesn't seem to be the same as in other tests in the repo.  Would it be possible to factor out into a helper file and use it consistently in all nock-based tests (or have comments explaining "this test can't use the standard helper for reason X") like `beforeEach(nockBeforeEach); afterEach(nockAfterEach)` or something?

Reply:
I like the suggestion, resolved via 13d09f0